src/Utilities/Extensions/ISymbolExtensions.cs

```diff
@@ -356,9 +356,11 @@ public static bool IsImplementationOfAnyImplicitInterfaceMember<TSymbol>(this IS
             return false;
         }
 
-        private static bool IsImplementationOfInterfaceMember(ISymbol symbol, ISymbol interfaceMember)
+        public static bool IsImplementationOfInterfaceMember(this ISymbol symbol, ISymbol interfaceMember)
         {
-            return interfaceMember != null && symbol.Equals(symbol.ContainingType.FindImplementationForInterfaceMember(interfaceMember));
+            return symbol != null &&
```

Comment:
It seems kind of deceiving if an extension method doesn't throw for a null `this` parameter. Above you could do

```cs
propertyReference.Property?.IsImplementationOfInterfaceMember(commandTextProperty) != true
```

instead.

Reply:
Fair. Seems like most of our extension methods need to have null checks and thow ArugmentNullExceptions added for the `this` parameter.